Modules/Loadable/Markups/VTKWidgets/vtkSlicerROIRepresentation3D.cxx

```diff
@@ -528,9 +528,6 @@ void vtkSlicerROIRepresentation3D::MarkupsInteractionPipelineROI::GetHandleColor
   double red[4]       = { 1.00, 0.00, 0.00, 1.00 };
   double green[4]     = { 0.00, 1.00, 0.00, 1.00 };
   double blue[4]      = { 0.00, 0.00, 1.00, 1.00 };
-  double orange[4]    = { 1.00, 0.50, 0.00, 1.00 };
```

Comment:
@Sunderlandkyl Where these colors intended to implement something else ?

Reply:
They can be deleted. I think that they were leftover from when I was experimenting with handle colors.